tests/platform/test_reboot.py

```diff
@@ -139,6 +149,16 @@ def reboot_and_check(localhost, dut, interfaces, reboot_type=REBOOT_TYPE_COLD, r
         logging.info("Further checking skipped for %s test which intends to verify reboot-cause only".format(reboot_type))
         return
 
+    check_further_interfaces_and_services(dut, interfaces)
+
+
+def check_further_interfaces_and_services(dut, interfaces):
+    """
+    Perform the further check after reboot-cause, including transceiver status, interface status
```

Comment:
suggest to  change to
'Perform a further check after reboot-cause including transceiver status, interface status'

Reply:
done.